src/main/java/com/google/cloud/tools/appengine/api/deploy/GenRepoInfoFileConfiguration.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.appengine.api.deploy;
+
+import java.nio.file.Path;
+
+/**
+ * Configuration for {@link GenRepoInfoFile#generate(GenRepoInfoFileConfiguration)}.
+ */
+public interface GenRepoInfoFileConfiguration {
+
+  Path getOutputDirectory();
```

Comment:
It's possible, but I think since the rest of the library relies on File, we should stick with it in this PR, and do a switch from File to Path globally as another PR, if everyone agrees on it.